config/filesystems.php

```diff
@@ -7,5 +7,9 @@
             'driver' => 'local',
             'root' => getcwd(),
         ],
+        'windowsconfig' => [
+            'driver' => 'local',
+            'root' => storage_path('../app/Commands/data'),
```

Comment:
We can probably simplify this with `app_path()`
```suggestion
            'root' => app_path('Commands/data'),
```